### ISSUE TITLE
Feature request: ACP entry for of array of always-shown channel IDs

### ISSUE THREAD

**Issue by Pohui-Nahui** [calzoneman/sync#867]
I think it would be nice if I had an ability to pin channels to the top even if they have no viewers

**Comment by antonpictures** [comment]
My name is George Anton and I have the same User Interface problem with my Calzoneman Website @ 
Under my account I have created the Start-Up Channels that I want PIN on the main page https://3reo.com , so I can share them with my family and friends.
3reo UI v1.0 Has to look at least like in this attached image:  https://i.imgur.com/q8cQy7g.jpg 
It is impossibile for me to bring people to use 3reo Calzoneman Website without the above pic desired UI look, because they don't understand the great functionality of Cytube so I want to lure them by "Hey, look Free TV channels on 3REO ! 
So many features that I want to add later. I hope Xaekai is ok. I hope he comes back.
If you know javascript and you are familiar with the code, please call me anytime.
Please help me PIN the channels on https://3reo.com Main Page

1 | http://3reo.com/chan/54 | 3reo MOVIES
2 | http://3reo.com/chan/56 | 3reo CINEMA
3 | http://3reo.com/chan/58 | ACTION
4 | http://3reo.com/chan/60 | COMEDY Movies
5 | http://3reo.com/chan/61 | SCI-FI Channel
6 | http://3reo.com/chan/62 | ROMANCE
7 | http://3reo.com/chan/64 | DRAMA
8 | http://3reo.com/chan/68 | HORROR
9 | http://3reo.com/chan/69 | FANTASY
10 | http://3reo.com/chan/70 | TERROR
11 | http://3reo.com/chan/72 | THRILLERS
12 | http://3reo.com/chan/74 | BLACK CINEMA
13 | http://3reo.com/chan/76 | 70's Theatre
14 | http://3reo.com/chan/79 | INDIES
15 | http://3reo.com/chan/84 | WESTERNS
16 | http://3reo.com/chan/86 | CLASSIC Movies
17 | http://3reo.com/chan/87 | 80's Theatre
18 | http://3reo.com/chan/88 | BW Movies
19 | http://3reo.com/chan/92 | CULT FILMS
20 | http://3reo.com/chan/94 | FLIX OF FURRY
21 | http://3reo.com/chan/96 | EURO Films
22 | http://3reo.com/chan/98 | 90's Theatre
23 | http://3reo.com/chan/111 | MTV 3reo
24 | http://3reo.com/chan/120 | ET Live
25 | http://3reo.com/chan/139 | BAYWATCH
26 | http://3reo.com/chan/151 | CELEBRITY
27 | http://3reo.com/chan/175 | BRITISH TV
28 | http://3reo.com/chan/221 | BBC News
29 | http://3reo.com/chan/222 | ABC News
30 | http://3reo.com/chan/223 | F24 News
31 | http://3reo.com/chan/224 | Sky News
32 | http://3reo.com/chan/225 | RT News
33 | http://3reo.com/chan/226 | CNA News
34 | http://3reo.com/chan/227 | AL JAZEERA News
35 | http://3reo.com/chan/232 | NBC News
36 | http://3reo.com/chan/247 | VOICE OF AMERICA
37 | http://3reo.com/chan/270 | NEWSMAX
38 | http://3reo.com/chan/308 | COPS
39 | http://3reo.com/chan/326 | SHARK TANK
40 | http://3reo.com/chan/350 | KUNG FU
41 | http://3reo.com/chan/351 | MAGIC
42 | http://3reo.com/chan/400 | 3reo SURPRISE
43 | http://3reo.com/chan/560 | 3reo DISCOVERY
44 | http://3reo.com/chan/563 | DISCOVERY SCIENCE
45 | http://3reo.com/chan/565 | 3reo HISTORY
46 | http://3reo.com/chan/567 | BIOGRAPHY
47 | http://3reo.com/chan/569 | MILITARY
48 | http://3reo.com/chan/571 | CONSPIRACY
49 | http://3reo.com/chan/573 | TRAVEL Channel
50 | http://3reo.com/chan/577 | ADVENTURE
51 | http://3reo.com/chan/579 | PSYCHICS
52 | http://3reo.com/chan/582 | ANIMAL Channel
53 | http://3reo.com/chan/611 | FOOD NETWORK
54 | http://3reo.com/chan/613 | BW Documentary
55 | http://3reo.com/chan/615 | CRIBS
56 | http://3reo.com/chan/633 | CHURCH
57 | http://3reo.com/chan/634 | JESUS TV
58 | http://3reo.com/chan/635 | FAITH TV
59 | http://3reo.com/chan/674 | ANIME WORLD
60 | http://3reo.com/chan/684 | CNET News
61 | http://3reo.com/chan/686 | IGN
62 | http://3reo.com/chan/701 | 3reo CARTOONS
63 | http://3reo.com/chan/709 | PEPA PIG
64 | http://3reo.com/chan/710 | SPONGE BOB
65 | http://3reo.com/chan/711 | FAMILY GUY
66 | http://3reo.com/chan/721 | DORA THE EXPLORER
67 | http://3reo.com/chan/796 | THE FAMOUS FIVE
68 | https://3reo.com/chan/800 | 3reo GEEKS
69 | http://3reo.com/chan/902 | Anton MUSIC TV
70 | http://3reo.com/chan/905 | LIVE CONCERTS
71 | http://3reo.com/chan/919 | GoodLife RADIO
72 | http://3reo.com/chan/920 | MindChill RADIO
73 | http://3reo.com/chan/921 | ChilledCow RADIO
74 | http://3reo.com/chan/922 | YellowBrick RADIO
75 | http://3reo.com/chan/923 | Relaxation RADIO
76 | http://3reo.com/chan/924 | BIG HTS RELEASED
77 | http://3reo.com/chan/927 | HIGHLINE R&B
78 | http://3reo.com/chan/930 | HIT LIST
79 | http://3reo.com/chan/952 | MAIN STAGE
80 | http://3reo.com/chan/953 | COUNTRY
81 | http://3reo.com/chan/955 | CRUCIFIX
82 | http://3reo.com/chan/956 | LATIN HITS
83 | https://3reo.com/chan/1000 | Joe Rogan TV
84 | http://3reo.com/chan/1001 | Howard Stern TV
85 | http://3reo.com/chan/1100 | 3reo SPORTS
86 | http://3reo.com/chan/1200 | 3reo SHOPPING
87 | http://3reo.com/chan/1201 | HSN
88 | http://3reo.com/chan/1202 | BEAUTY IQ
89 | http://3reo.com/chan/1206 | QVC

**Comment by calzoneman** [comment]
Can you clarify if you are asking from the perspective of a server owner or an individual user (which would be similar to https://github.com/calzoneman/sync/issues/742 I guess)?

**Comment by Pohui-Nahui** [comment]
> Can you clarify if you are asking from the perspective of a server owner or an individual user (which would be similar to #742 I guess)?

i mean global pin, so from the server owner perspective

**Comment by Xaekai** [comment]
I may implement this just because I'm tired of how many times it's come up.

Whenever people come around asking how to turn CyTube into some sort of television network, (regardless of how much we've made it clear that wasn't the intent for this software they still keep appearing), this is invariably one of the things they require.

**Comment by antonpictures** [comment]
> 
> 
> Can you clarify if you are asking from the perspective of a server owner or an individual user (which would be similar to #742 I guess)?
Why was my comment deleted?

**Comment by Xaekai** [comment]
@antonpictures:
Because it was a large block of information that wasn't specifically relevant here. It was inappropriate off-topic spam for the sync server issue tracker.

**Comment by antonpictures** [comment]
> 
> 
> @antonpictures:
any updates on indexing the channels/favs with the youtube embed on the main page please?

**Comment by calzoneman** [comment]
Is that related to the feature request in this issue or something else?

Regardless, I don't have plans to work on it, but I left the issue open in
case someone wants to submit a PR

**Comment by antonpictures** [comment]
Yes. I want to share with my facebook friends the channels I have created on the main page. An option   to add channels as favorites to be shown on the main page https://3reo.com, at least would be great to be made by you as the Creator.

**Comment by calzoneman** [comment]
I just want to set the expectation clearly that I don't plan to implement this.  I'm not trying to be rude, nor to imply that what anyone is suggesting is a bad idea, but a lot of such feature requests don't align with my goals for the project.

It is my personal philosophy for this project that I would prefer to maintain a relatively light core which interested users can fork and customize to their needs, rather than accepting a large number of features (whether or not I personally implement them or they come via PR) and associated configuration.  Several users have taken advantage of this and many CyTube-derived sites exist across the internet with various customizations relevant to each community's individual needs.

Separately, my life has changed a lot since I started this project and I no longer have the free time and energy to implement a significant amount of feature work.  I continue to maintain it and implement minor bugfixes and improvements, but I'm generally not taking on new features myself.  I am willing to review PRs and some people have contributed, but PRs also need to consider the above philosophy.

I will take an action item to update the issue templates to make this expectation more clear when users submit an issue.

**Comment by Xaekai** [comment]
https://www.theolognion.com/open-source-maintainer-apologizes-for-having-a-day-time-job/
